pkg/loadbalancer/reconciler/termination.go

```diff
@@ -239,13 +239,15 @@ func terminateUDPConnectionsToBackend(p socketTerminationParams, l3n4Addr lb.L3n
 	// Iterate over all pod network namespaces, and terminate any stale connections.
 	if p.ExtConfig.EnableSocketLBPodConnectionTermination && !p.ExtConfig.BPFSocketLBHostnsOnly {
 		iter, errs := p.NetNSOps.all()
-		for name, ns := range iter {
-			destroy(name, ns)
-		}
 		for err := range errs {
 			p.Log.Debug("Error opening netns, skipping",
 				logfields.Error, err)
 		}
+		if iter != nil {
```

Comment:
This does not work. We must iterate over `iter` first as otherwise we'll block forever in `for err := range errs`.